### PR TITLE
Migrate to GitHub Issue Forms

### PR DESCRIPTION
**Pull request by gastaldi**
This migrates the existing issue templates to [GitHub Issue Forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)

### CODE REVIEW

**Comment by gastaldi** [comment]
Here is a preview in case you're wondering how it would look like: 

https://github.com/gastaldi/quarkus/blob/issue_forms/.github/ISSUE_TEMPLATE/bug_report.yml

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      label: Actual behavior
+      description: >-
+        Describe the actual behavior clearly and concisely.
+  - type: textarea
+    attributes:
+      label: How to Reproduce?
+      description: >-
+        Link to a small reproducer (preferably a Maven project if the issue is not Gradle-specific) or attach an archive containing the reproducer to the issue.
+      placeholder: |
+        Steps to reproduce the behavior:
```
Comment (gsmet) [inline .github/ISSUE_TEMPLATE/bug_report.yml:31]: Can you add `Reproducer: ` before the steps with a new line? We have so many bug reports without reproducer, I would like to make it very explicit.

**Review by gsmet — COMMENTED** [review]
Looks good, thanks for doing that. I added a small suggestion.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (gastaldi) [reply .github/ISSUE_TEMPLATE/bug_report.yml:31]: That's a valid concern. Should we make this field mandatory?

**Comment by gastaldi** [comment]
Let me know if anything else is missing 😉

**Review by gsmet — APPROVED** [review]
OK, let's try that and see how it works.

Thanks for taking care of it, it must have been a bit tedious.

**Pull request merged**